typescript/packages/well-log-viewer/src/utils/well-log.ts

```diff
@@ -0,0 +1,119 @@
+import _ from "lodash";
+import type {
+    WellLogCurve,
+    WellLogMetadataDiscrete,
+    WellLogSet,
+} from "../components/WellLogTypes";
+import type { AxesInfo } from "./tracks";
+import type { WellLogViewProps } from "../components/WellLogView";
+
+export function getAllWellLogCurves(wellLog: WellLogSet[]): WellLogCurve[] {
```

Comment:
Do you mind writing some doc and unit tests for these functions ?

Reply:
Will do